src/lib/models/event-history/get-event-billable-actions.ts

```diff
@@ -0,0 +1,64 @@
+import type { WorkflowEvent } from '$lib/types/events';
+import { isWorkflowTaskFailedEventDueToReset } from '$lib/utilities/get-workflow-task-failed-event';
+import {
+  isActivityTaskScheduledEvent,
+  isActivityTaskStartedEvent,
+  isMarkerRecordedEvent,
+  isNexusOperationCancelRequestedEvent,
+  isNexusOperationScheduledEvent,
+  isSignalExternalWorkflowExecutionInitiatedEvent,
+  isStartChildWorkflowExecutionInitiatedEvent,
+  isTimerStartedEvent,
+  isUpsertWorkflowSearchAttributesEvent,
+  isWorkflowExecutionContinuedAsNewEvent,
+  isWorkflowExecutionOptionsUpdatedEvent,
+  isWorkflowExecutionSignaledEvent,
+  isWorkflowExecutionStartedEvent,
+  isWorkflowExecutionUpdateAcceptedEvent,
+  isWorkflowExecutionUpdateRejectedEvent,
+  isWorkflowTaskCompletedEvent,
+} from '$lib/utilities/is-event-type';
+
+export const getEventBillableActions = (event: WorkflowEvent): number => {
+  try {
+    if (isWorkflowExecutionStartedEvent(event)) return 1;
+    if (isActivityTaskScheduledEvent(event)) return 1;
+    if (isTimerStartedEvent(event)) return 1;
+    if (isWorkflowExecutionSignaledEvent(event) && event.id !== '2') return 1;
```

Comment:
Maybe a comment here explaining why we exclude signal events if the id is `2` would be good for future us.